README.md

```diff
@@ -82,13 +82,19 @@ The primary supported way of running PeopleMath in a staging or production confi
 * Run `build_appengine.sh` or equivalent commands to build the front-end and generate the `appengine_dist` directory
 * `cd appengine_dist` and `gcloud app deploy`
 
-### Authentication
+### Authentication & Authorization
 
-To enable authentication using Firebase:
+A basic authorization mechanism has been implemented which specifies one email domain (like "google.com").
+Every user with an email address ending in that domain has read and write access.
+Any other user will not have access.
+
+To enable authorization with Firebase to authenticate users:
```

Comment:
Nit: can you link to the Firebase Auth site here?

https://firebase.google.com/docs/auth